infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/fsm/LogReplicationEvent.java

```diff
@@ -51,6 +52,9 @@ public enum LogReplicationEventType {
      */
     private LogReplicationEventMetadata metadata;
 
+    @Getter
+    private LogReplicationFSM replicationFsm;
```

Comment:
is it required now?

Reply:
No, i missed this...thought had removed this one!